### DISCUSSION TITLE
Add Chapter/Bookmark View Button

### DISCUSSION THREAD

**Discussion by MPaandeey** [huynhsontung/Screenbox#508] [Ideas]
I would like to request the addition of a "Chapter View" button to the media player. This feature would enhance the user experience by allowing viewers to easily navigate through different sections or chapters of a video tutorial.
![Screenshot 2024-12-14 165338](https://github.com/user-attachments/assets/aa3eddd1-6d6f-4268-8bd0-525c9e85ac0c)

**Comment by United600** [comment]
Great suggestion. We should eventually show a context menu that includes the chapter list.

Here is a quick mock-up to use as a reference for the implementation:
<img width="1194" alt="Chapters Context Menu" src="https://github.com/user-attachments/assets/fb965b81-e4a4-47e1-a4ac-cdc82f682a42" />

**Reply by MPaandeey** [reply]
Thank you for considering this feature!
